features/command_line_interface/options.feature

```diff
@@ -41,7 +41,8 @@ Feature: Reek can be controlled using command-line options
 
       Configuration:
           -c, --config FILE                Read configuration options from FILE
-              --smell SMELL                Detect smell SMELL (default: all enabled smells)
+              --smell SMELL                Detect smell or smell category SMELL
+                                             (default: all enabled smells)
```

Comment:
Also, can we add some examples for both (smell types and smell categories) to our README including helpful links to our docs so the user can explore that on its own?
If we just put it out like this I believe the mental barrier for the user is so high that nobody except for maybe us will use it...